jenkins/Jenkinsfile

```diff
@@ -0,0 +1,477 @@
+#!/bin/groovy
+
+// global variables
+repository = "xamarin/xamarin-macios"
+isPr = false
+branchName = null
+gitHash = null
+packagePrefix = null
+virtualPath = null
+xiPackageUrl = null
+xmPackageUrl = null
+utils = null
+errorMessage = null
+currentStage = null
+
+xiPackageFilename = null
+xmPackageFilename = null
+manifestFilename = null
+reportPrefix = null
+createFinalStatus = true
+skipLocalTestRunReason = ""
+
+def abortExecutingBuilds ()
```

Comment:
Since this is all commented, do we need it in the diff? Or should it be deleted?

Reply:
It will be working pretty soon (needs some changes to Jenkins itself first), but I can remove it.